databases/week1/database_week1.sql

```diff
@@ -0,0 +1,65 @@
+CREATE database_week1
+    DEFAULT CHARACTER SET = "utf8mb4";
+-- Find out how many tasks are in the task table
+SELECT 
+COUNT(id) as task_amount 
+FROM 
+task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id) 
+from task 
+WHERE due_date is null;
+
+-- Find all the tasks that are marked as done
+SELECT *
+FROM task 
+JOIN status
+on task.status_id = status.id
+where name = "done";
+
+-- Find all the tasks that are not marked as done
+SELECT *
+FROM task 
+JOIN status
+ON task.status_id = status.id
+WHERE name != "done";
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT * 
+FROM task
+ORDER BY due_date DESC;
+
+-- Get the single most recently created task
+SELECT MAX(created)
+FROM task;
```

Comment:
This will only retrieve the creation date - but what if you'd want to get all the info about this task? Hint: you could select every column from the task table, order it and limit the results to 1 😄